data/raw/keybindings.json

```
@@ -3428,5 +3428,12 @@
     "id": "CHECK_OPINION",
     "name": "Check opinion",
     "bindings": [ { "input_method": "keyboard_char", "key": "O" }, { "input_method": "keyboard_code", "key": "o", "mod": [ "shift" ] } ]
+  },
+  {
+    "type": "keybinding",
+    "category": "FACTION_MANAGER",
+    "id": "INSPECT_NPC",
+    "name": "Inspect NPC",
+    "bindings": [ { "input_method": "keyboard_char", "key": "i" } ]
```

```suggestion
    "bindings": [ { "input_method": "keyboard_any", "key": "i" } ]
```
Add keybindings for both symbol and keycode input modes.